pkg/common/client_state.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import (
+	"context"
+
+	kc "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"github.com/keycloak/keycloak-operator/pkg/model"
+	v1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ClientState struct {
+	Client       *kc.KeycloakAPIClient
+	ClientSecret *v1.Secret
+	Context      context.Context
+	Keycloak     *kc.Keycloak
```

Comment:
I can't find this used anywhere. Should be able to remove unless I am missing something.

Reply:
Fixed.